frontend/src/types/Ambient.tsx

```diff
@@ -5,6 +5,7 @@ export enum WaypointType {
   Workload = 'workload'
 }
 
+export const Waypoint = 'waypoint';
 export const waypintForLabel = 'istio.io/waypoint-for';
```

Comment:
```suggestion
export const WaypointForLabel = 'istio.io/waypoint-for';
```

typo, but also made capital since the other constants also seem to be capitalized.